examples/vae/ss_vae_M2.py

```diff
@@ -388,29 +395,27 @@ def run_inference_ss_vae(args):
         visualize(ss_vae, viz, data_loaders["test"])
     finally:
         # close the logger file object if we opened it earlier
-        if args.logfile is not None:
+        if args.logfile:
             logger.close()
 
 
-EXAMPLE_RUN = "example run: python example_M2.py --seed 0 -cuda -ne 2 --aux-loss -alm 300 -enum -sup 3000 " \
+EXAMPLE_RUN = "example run: python ss_vae_M2.py --seed 0 -cuda -ne 2 --aux-loss -alm 300 -enum -sup 3000 " \
               "-zd 20 -hl 400 200 -lr 0.001 -b1 0.95 -bs 500 -eps 1e-7 -log ./tmp.log"
 
 if __name__ == "__main__":
-    import argparse
 
-    parser = argparse.ArgumentParser(description="SS-VAE model inference\n{}".format(EXAMPLE_RUN))
+    parser = argparse.ArgumentParser(description="SS-VAE\n{}".format(EXAMPLE_RUN))
 
     parser.add_argument('-cuda', '--use-cuda', action='store_true',
                         help="use GPU(s) to speed up training")
-    parser.add_argument('-ne', '--num-epochs', default=1, type=int,
+    parser.add_argument('-ne', '--num-epochs', default=100, type=int,
                         help="number of epochs to run")
     parser.add_argument('--aux-loss', action="store_true",
                         help="whether to use the auxiliary loss from NIPS 14 paper (Kingma et al)")
     parser.add_argument('-alm', '--aux-loss-multiplier', default=300, type=float,
                         help="the multiplier to use with the auxiliary loss")
     parser.add_argument('-enum', '--enum-discrete', default=None,
-                        help="whether to enumerate the discrete support of the categorical distribution"
-                             "while computing the ELBO loss")
+                        help="parallel or sequential")
```

Comment:
does this mean sampling the discrete is not an option?

Reply:
If you leave it blank or set it to the empty string, then discretes should be sampled. @jpchen can you indicate this in the help message?